sdk/ai/azure-ai-client/samples/agents/sample_agents_basics.py

```diff
@@ -29,32 +32,24 @@
     logging_enable=True, # Optional. Remove this line if you don't want to show how to enable logging
 )
```

Comment:
Please do not change the existing samples, was there a reason?

Reply:
Are you talking about the FILE, DESCRIPTION and USAGE?   I think Darron didn't include it in his first agent basic sample, so I follow the pattern.   But he just mentioned to add them.   So now I have them.

Reply:
According to our call, I have resolved this.